### DISCUSSION TITLE
Entrance information

### DISCUSSION THREAD

**Discussion by PetroTruemetrics** [OvertureMaps/data#237] [Discussion]
Hi! I am trying to find a way of extracting details about the buildings, namely entrance information and location. I used "place" type and theme, but it doesn't always or almost never matches the actual entrance of a business/place. 

Is there a better way of getting that information from Overture data or are there any plans for improving the building details?

**Comment by jenningsanderson** [comment]
Great question @PetroTruemetrics. The short answer is _yes_, we will continually improve the details available for buildings. 

In the meantime, does OSM have the building entrance information you are looking for? https://wiki.openstreetmap.org/wiki/Key:entrance
